options/core/src/main/java/polanski/option/OptionAssertion.java

```diff
@@ -0,0 +1,63 @@
+package polanski.option;
+
+import android.support.annotation.NonNull;
+
+import polanski.option.function.Func1;
+
+public final class OptionAssertion<T> {
```

Comment:
Perhaps we could also expose some predicates `isNone` etc to make testing with Rx more streamlined. Although, I guess that wouldn't really work because it would require this library to use Rx predicate function classes.